documentation/book/con-tls-within-the-cluster.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='tls-within-the-cluster-{context}']
+= TLS within the cluster
+
+All the internal communication within the Kafka cluster is encrypted with TLS. 
+
+== Zookeeper Communication
+
+Zookeeper does not support TLS itself. 
+By deploying an `stunnel` sidecar within every Zookeeper pod, the Cluster Operator is able to provide data encryption and authentication between Zookeeper nodes in a cluster.
+Zookeeper communicates only with the `stunnel` sidecar over the loopback interface.
+The `stunnel` sidecar then proxies all Zookeeper traffic, TLS decrypting data upon entry into a Zookeeper pod and TLS encrypting data upon departure from a Zookeeper pod.
+
+This TLS encrypting `stunnel` proxy is instantiated from the `spec.zookeeper.stunnelImage` specified in the Kafka resource.
+
+== Interbroker Kafka Communication
+
+Communication between Kafka brokers is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
+
+Communication between Kafka brokers and Zookeeper nodes uses an `stunnel` sidecar, as described above.
+
+== Topic and User Operators
+ 
+ The Topic and User Operators each use an `stunnel` sidecar, as described above when communicating with Zookeeper.
```

Comment:
We want to avoid saying 'above' or 'below' when referencing other sections because we can't be sure that the user is reading in order.  I think it's safe to 'remove' and maybe say 'Like the Cluster Operator, the Topic and User Operators....'